packages/edit-site/src/components/page-templates/dataviews-templates.js

```diff
@@ -0,0 +1,197 @@
+/**
+ * External dependencies
+ */
+import removeAccents from 'remove-accents';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	__experimentalHeading as Heading,
+	__experimentalText as Text,
+	__experimentalVStack as VStack,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { useState, useMemo, useCallback, useEffect } from '@wordpress/element';
+import { useEntityRecords } from '@wordpress/core-data';
+import { decodeEntities } from '@wordpress/html-entities';
+import { privateApis as routerPrivateApis } from '@wordpress/router';
+
+/**
+ * Internal dependencies
+ */
+import Page from '../page';
+import Link from '../routes/link';
+import AddedBy from '../list/added-by';
+import { TEMPLATE_POST_TYPE } from '../../utils/constants';
+import { DataViews } from '../dataviews';
+import DEFAULT_VIEWS from './default-views';
+import { useResetTemplateAction } from './template-actions';
+import { unlock } from '../../lock-unlock';
+
+const { useLocation } = unlock( routerPrivateApis );
+
+const EMPTY_ARRAY = [];
+const defaultConfigPerViewType = {
+	list: {},
+	grid: {},
+};
+
+function normalizeSearchInput( input = '' ) {
+	return removeAccents( input.trim().toLowerCase() );
+}
+
+export default function DataviewsTemplates() {
+	const {
+		params: { path, activeView = 'all' },
+	} = useLocation();
+	const initialView = DEFAULT_VIEWS.find(
+		( { slug } ) => slug === activeView
+	).view;
+	const [ view, setView ] = useState( initialView );
+	useEffect( () => {
+		setView(
+			DEFAULT_VIEWS.find( ( { slug } ) => slug === activeView ).view
+		);
+	}, [ path, activeView ] );
```

Comment:
Do we need all this logic, do we expect support for multiple global views as well here. Might be good to get some designers on this PR :)

Reply:
Agreed. For the first iteration it should be better to remove it.